plugins/starboard.yaml

```diff
@@ -0,0 +1,73 @@
+apiVersion: krew.googlecontainertools.github.com/v1alpha2
+kind: Plugin
+metadata:
+  name: starboard
+spec:
+  version: "v0.2.1"
+  homepage: https://github.com/aquasecurity/starboard
+  shortDescription: >-
+    Kubernetes-native security tool kit
+  description: |+2
+    Initialize the starboard(create the custom resources, ns, sa):
+    $ kubectl starboard init
+    
+    After init startboard, you could interact with starboard following this structure:
+    starboard [VERB] TOOL [TYPE | TYPE/NAME | NONRESOURCEURL]
+
+    TOOL is the security tool in the cloud native like kube-hunter, kube-bench, vuln(trivy)
+    VERB is a logical Kubernetes API verb like 'get', 'find', etc.
+    TYPE is a Kubernetes resource. Shortcuts and API groups will be resolved, e.g. 'po' or 'pod.metrics.k8s.io'.
+    NAME is the name of a particular Kubernetes resource.
+    NONRESOURCEURL is a partial URL that starts with "/".
```

Comment:
ideally this part as well should appear in your --help.
use this `description:` field to explain capabilities of your plugin.

also make sure when run as a plugin your --help message shows kubectl prefix.

Reply:
@ahmetb Thank you for pointing this out. Starbord CLI v0.2.1 does not recognize whether it runs as a standalone binary or a kubectl plugin. Therefore some usage examples might not use kubectl prefix. However we're aware of this limitation and we're going to fix that asap in the upcoming releases.

In this PR our goal is to add Starboard to the index, so that we could setup krew-release-bot and automate upgrades, which will include enhancement such as help messages and other features.